app/models/like.rb

```diff
@@ -0,0 +1,8 @@
+class Like < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :comment
```

Comment:
- [x] According to the [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png) The belongs-to relationship of Likes is with a Post, not a comment, please consider fixing this one! 🤓 🔧 

```suggestion
  belongs_to :post
```

Reply:
Thanks @anagudelogu I'll make the changes